public/app/plugins/datasource/elasticsearch/query_builder.ts

```diff
@@ -493,7 +497,7 @@ export class ElasticQueryBuilder {
     return query;
   }
 
-  getLogsQuery(target: ElasticsearchQuery, limit: number, adhocFilters?: any, querystring?: string) {
+  getLogsQuery(target: ElasticsearchQuery, limit: number, adhocFilters?: any) {
```

Comment:
like for the `build` method, the last param is removed as it was always being set from `target`